src/lib389/lib389/nss_ssl.py

```diff
@@ -1271,6 +1277,14 @@ def add_ca_cert_bundle(self, cert_file, nicknames):
                                 names_len = len(nicknames)
                                 if names_len > ca_count:
                                     if nicknames[ca_count].lower() == CERT_NAME.lower() or nicknames[ca_count].lower() == CA_NAME.lower():
+                                        # cleanup
+                                        try:
+                                            for tmp_file in ca_files_to_cleanup:
+                                                os.remove(tmp_file)
+                                        except IOError as e:
+                                            # failed to remove tmp cert
+                                            log.debug("Failed to remove tmp cert file: " + str(e))
+                                            pass
```

Comment:
I don't think `pass` is needed here as we have the `log.debug` command and there's no need in a placeholder statement.